lib/board.rb

```diff
@@ -28,7 +28,22 @@ def valid_coordinate?(coordinate)
     false
   end
 
-  def valid_placement?(ship, array)
+  def valid_placement_length?(ship, array)
     return true if ship.length == array.length
+    # require "pry"; binding.pry
+  end
+
+  def valid_placement_consecutive?(ship, array)
+    cell_1 = array[0].split''
+    cell_2 = array[1].split''
+    number_1 = cell_1[1].to_i
```

Comment:
Splitting up the methods is a good move. Should allow us to pass the consecutive test without failing the length test.